ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -745,7 +751,9 @@ static KsqlRestApplication buildApplication(
     final ConcurrencyLimiter pullQueryConcurrencyLimiter = new ConcurrencyLimiter(
         ksqlConfig.getInt(KsqlConfig.KSQL_QUERY_PULL_MAX_CONCURRENT_REQUESTS_CONFIG),
         "pull queries");
-
+    final SlidingWindowRateLimiter pullBandRateLimiter = new SlidingWindowRateLimiter(
+            ksqlConfig.getInt(KsqlConfig.KSQL_QUERY_PULL_MAX_HOURLY_BANDWIDTH_CONFIG),
+            3600 * 1000);
```

Comment:
`3600 * 1000` might be good to either put in a constant here or maybe even in another config if we think we might adjust how this is enforced.